hivemind/averaging/matchmaking.py

```diff
@@ -521,8 +512,6 @@ async def _declare_averager_periodically(self, key_manager: GroupKeyManager):
                     await asyncio.sleep(self.declared_expiration_time - get_dht_time())
                     if self.running.is_set() and len(self.leader_queue) == 0:
                         await key_manager.update_key_on_not_enough_peers()
-            except (concurrent.futures.CancelledError, asyncio.CancelledError):
-                pass  # note: this is a compatibility layer for python3.7
```

Comment:
[non-blocking]
Q: are you certain that we no longer need to handle concurrent.futures.CancelledError  here or is it an educated guess?

Reply:
I have edited this PR to only remove the `except` statements in `_update_queue_periodically()` and `_declare_averager_periodically()` methods.

These methods are only awaited in `cancel_and_wait()` that handles both cancel and normal errors by itself.